app/serializers/comment_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class CommentSerializer < ActiveModel::Serializer
+  include TagHelper
+
+  attributes :id, :content, :markdown, :commentable_type, :commentable_id, :owner_id, :created_at, :updated_at, :team_id
```

Comment:
Right now I don’t think so as we’ll be just grabbing comments for a story from `/projects/:id/stories/:id/comments` but we may need to rethink this when we make comments available on everything. It just includes the owner so we can display their name / avatar.